src/pages/household/output/NetIncomeBreakdown.jsx

```diff
@@ -8,16 +9,66 @@ import {
 import ResultsPanel from "../../../layout/ResultsPanel";
 import style from "../../../style";
 
+// The arrows are used to differentiate increases or decreases
+// to net income for colorblind users.
+function UpArrow() {
+  return (
+    <span
+      style={{
+        display: "inline-block",
+        fontWeight: "bold",
+        fontSize: "1.5em",
+        color: style.colors.DARK_GREEN,
+      }}
+    >
+      &uarr;
+    </span>
+  );
+}
+
+function DownArrow() {
```

Comment:
I like this UI feature, nice- could we adjust the alignment slightly though? So it's inline and (if not non-trivial) same-height with the text: currently looks like it's vertically off a bit.